wrapanapi/systems/msazure.py

```diff
@@ -1111,3 +1143,15 @@ def is_stack_empty(self, stack_name, resource_group):
 
     def info(self):
         pass
+
+    def cleanup_resources(self):
+        """
+        Clean up Unused Disks, Nics and Public IPs
+
+        Any exceptions raised during NIC/PIP/Disc delete attempts are logged only.
+        """
+        self.logger.info("cleanup: Removing all unused NICs/PIPs/Discs in resource group")
+        self.remove_nics_by_search(self.resource_group)
+        self.remove_discs_by_search(self.resource_group)
+        self.remove_pips_by_search(self.resource_group)
+        return True
```

Comment:
This return is unnecessary as it doesn't actually indicate anything to the caller.

Reply:
I copied the same behavior from another VM Cleanup function in this module :)

I can remove as well!

Reply:
Removed !